<?xml version="1.0" encoding="UTF-8"?><testrun name="Rerun com.test.DeptServiceTest.initializationError" project="15Day_TDD_MVC_@SQL2" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="initializationError" classname="com.test.DeptServiceTest" time="0.0">
    <error>org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.test.context.web.WebDelegatingSmartContextLoader]: Constructor threw exception; nested exception is java.lang.NoClassDefFoundError: javax/servlet/ServletContext&#13;
	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:154)&#13;
	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:102)&#13;
	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:125)&#13;
	at org.springframework.test.context.support.AbstractTestContextBootstrapper.resolveContextLoader(AbstractTestContextBootstrapper.java:474)&#13;
	at org.springframework.test.context.support.AbstractTestContextBootstrapper.buildMergedContextConfiguration(AbstractTestContextBootstrapper.java:359)&#13;
	at org.springframework.test.context.support.AbstractTestContextBootstrapper.buildMergedContextConfiguration(AbstractTestContextBootstrapper.java:305)&#13;
	at org.springframework.test.context.support.AbstractTestContextBootstrapper.buildTestContext(AbstractTestContextBootstrapper.java:112)&#13;
	at org.springframework.test.context.TestContextManager.&lt;init&gt;(TestContextManager.java:120)&#13;
	at org.springframework.test.context.TestContextManager.&lt;init&gt;(TestContextManager.java:105)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTestContextManager(SpringJUnit4ClassRunner.java:152)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.&lt;init&gt;(SpringJUnit4ClassRunner.java:143)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.internal.builders.AnnotatedBuilder.buildRunner(AnnotatedBuilder.java:104)&#13;
	at org.junit.internal.builders.AnnotatedBuilder.runnerForClass(AnnotatedBuilder.java:86)&#13;
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#13;
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#13;
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#13;
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:33)&#13;
	at org.junit.internal.requests.FilterRequest.getRunner(FilterRequest.java:36)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.createFilteredTest(JUnit4TestLoader.java:77)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.createTest(JUnit4TestLoader.java:68)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.loadTests(JUnit4TestLoader.java:43)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:444)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.lang.NoClassDefFoundError: javax/servlet/ServletContext&#13;
	at org.springframework.test.context.web.WebDelegatingSmartContextLoader.&lt;init&gt;(WebDelegatingSmartContextLoader.java:63)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:142)&#13;
	... 28 more&#13;
Caused by: java.lang.ClassNotFoundException: javax.servlet.ServletContext&#13;
	at java.net.URLClassLoader.findClass(Unknown Source)&#13;
	at java.lang.ClassLoader.loadClass(Unknown Source)&#13;
	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)&#13;
	at java.lang.ClassLoader.loadClass(Unknown Source)&#13;
	... 34 more&#13;
&#13;
</error>
  </testcase>
</testrun>
